### PR TITLE
More granularly define bounty payouts.

### PR DESCRIPTION
**Pull request by tomrittervg**
@dveditz @mozfreddyb

### CODE REVIEW

**Review by mozfreddyb — APPROVED** [review]

**Review by dveditz — APPROVED** [review]
great changes, thanks!

**Comment by mozfreddyb** [comment]
@stevejalim FYI

File: bedrock/security/templates/security/client-bug-bounty.html [hunk 1]
```diff
@@ -82,16 +82,28 @@ <h4 id="security-vulnerability-amounts">Rewards Amount</h4>
...
+    <!-- ========================================================= -->
     <tr>
       <td>Highest Impact
-        <ul class="mzp-u-list-styled">
+      </td>
+      <td>$20,000</td>
+      <td>$18,000</td>
+    </tr>
+    <tr>
+      <td style="border-top: 0px">
```
Comment (alexgibson) [inline bedrock/security/templates/security/client-bug-bounty.html:93]: Please remove inline CSS styles from this PR. CSS should be in `.scss` files, external to a page's HTML.

**Review by alexgibson — CHANGES_REQUESTED** [review]

**Comment by craigcook** [comment]
Closed in favor of https://github.com/mozilla/bedrock/pull/14614. Thanks!

**Pull request closed without merging**